components/AppBar/tests/unit/AppBarWrappingLegacyTopLayoutGuideTests.swift

```diff
@@ -0,0 +1,151 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import XCTest
+import MaterialComponents.MaterialAppBar
+import MaterialComponents.MaterialUIMetrics
+
+class AppBarWrappingLegacyTopLayoutGuideTests: XCTestCase {
+
+  // MARK: No scroll view
+
+  func testNoScrollViewTopLayoutGuideEqualsZero() {
+    // Given
+    let contentViewController = UIViewController()
+
+    let container = MDCAppBarContainerViewController(contentViewController: contentViewController)
+    container.isTopLayoutGuideAdjustmentEnabled = false
+    container.appBar.headerViewController.headerView.minMaxHeightIncludesSafeArea = false
+    container.appBar.headerViewController.headerView.minimumHeight = 50
+    container.appBar.headerViewController.headerView.maximumHeight = 100
+    let _ = container.view // Force the view to load.
+
+    // Then
+    XCTAssertEqual(contentViewController.topLayoutGuide.length, 0)
+    #if swift(>=3.2)
```

Comment:
Suggestion:  Do we want to ensure that the topLayoutGuide's view controller is nil?
XCTAssertNil(container.appBar.headerViewController.topLayoutGuideViewController)

I'm not sure why we modify the property, but it seems important if we're doing it.

Reply:
Done.